src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java

```diff
@@ -580,6 +600,24 @@ public void close() throws IOException {
         eventLoopGroup.shutdownGracefully();
     }
 
+    /**
+     * Configure connection timeout in milliseconds
+     */
+    public NettyDockerCmdExecFactory withConnectTimeout(Integer connectTimeout) {
```

Comment:
this option is must have for any remote connection and we can remove it any time later.